.github/workflows/release.yml

```
@@ -99,7 +99,7 @@ jobs:
           repository: infrahq/web
           token: ${{ secrets.DOCS_TOKEN }}
       - run: |
-          echo $GITHUB_REF  > ./docs-ref
+          echo $GITHUB_SHA  > ./docs-ref
```

maybe `tee` here so it also prints out the input?

```suggestion
          echo $GITHUB_SHA  | tee ./docs-ref
```